<launch>

    <include file="$(find crues_control)/robot.launch" />

    <node name="roomba" pkg="crues_control" type="roomba_control.py">
        <param name="turn_vel" type="double" value="2" />
        <param name="fwd_vel" type="double" value="0.2" />
        <param name="obstacle_range" type="double" value="100" />
        <remap from="~rate" to="enc_rate" />
    </node>

</launch>